src/dbg/_plugins.h

```diff
@@ -256,6 +261,7 @@ typedef enum
 {
     CB_INITDEBUG, //PLUG_CB_INITDEBUG
     CB_STOPDEBUG, //PLUG_CB_STOPDEBUG
+    CB_STOPPINGDEBUG, //PLUG_CB_STOPDEBUG

```

Comment:
You cannot add the enum here because it will break binary compatibility

Reply:
Correct, forgot about that. Moved the enum to the bottom, before `CB_LAST`.